tsconfig.json

```diff
@@ -1,7 +1,10 @@
 {
   "extends": "./bazel.tsconfig.json",
+  "compilerOptions": {
+    "noEmit": true
```

Comment:
This is helpful for development, since it prevents you from accidentally writing a massive `.js` and `.js.map` file tree